### DISCUSSION TITLE
Welcome to cfdtunnel Discussions!

### DISCUSSION THREAD

**Discussion by mmiranda** [mmiranda/cfdtunnel#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jeniphan** [comment]
Hi @mmiranda. I try use your product on Raspberry pi with Debian 11 maybe. I installed cfdtunnel from brew and cloudflared also. If I need run command "cloudflared access tcp --hostname yyy.mmm.com --url localhost:3366" How write file config and run command? you do have example? Sorry for language. Thanks for help.
